frontend/src/components/sidemenu/sidemenu.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import { Sidemenu } from '.';
+
+describe('Sidemenu', () => {
+  it('should render sidemenu and check system name in screen', () => {
```

Comment:
Usually it has no ands so I suggest 
"Should check system name"

Reply:
Thanks! I go todo this.